apps/laboratory/tests/shared/pages/ModalWalletPage.ts

```diff
@@ -36,7 +36,8 @@ export class ModalWalletPage extends ModalPage {
 
   override async switchNetwork(network: string) {
     await this.page.getByTestId('account-switch-network-button').click()
-    await this.page.getByTestId(`w3m-network-switch-${network}`).click()
+    const networkToSwitchButton = this.page.getByTestId(`w3m-network-switch-${network}`)
+    await networkToSwitchButton.click()
```

Comment:
the change is needed?

Reply:
maybe not, it's a leftover from a debug session